src/apps/reservation-list/reservation-material/reservation-material.tsx

```diff
@@ -15,13 +21,37 @@ const ReservationMaterial: FC<ReservationMaterialProps & MaterialProps> = ({
   material,
   reservation
 }) => {
-  const openDetailsModal = useCallback((e: MouseEvent) => {
-    e.stopPropagation();
-    // Todo
-  }, []);
+  const { open } = useModalButtonHandler();
+  const config = useConfig();
+
+  // Get library branches from config
+  const inputBranches = config<AgencyBranch[]>("branchesConfig", {
+    transformer: "jsonParse"
+  });
+
+  // Get the library branches where the user cannot pick up books at
+  const blacklistBranches = config("blacklistedPickupBranchesConfig", {
+    transformer: "stringToArray"
+  });
+
```

Comment:
Great that you use the new data prop config system.
I know that Kasper B. has made some changes to this in the release/4 branch.
Something with overload functions. I am not sure if it will affect this.
But now you know it.